File: week2/p5.c
#include <stdio.h>

int main() {
    int m = 3, n = 3;
    int mat[3][3] = {
        {1, 0, 3},
        {0, 0, 4},
        {6, 0, 0}
    };

    int zero_count = 0;
    for (int i = 0; i < m; i++) {
        for (int j = 0; j < n; j++) {
            if (mat[i][j] == 0) {
                zero_count++;
            }
        }
    }

    if (zero_count > (m * n) / 2)
        printf("Yes\n");
    else
        printf("No\n");

    return 0;
}

File: week2/p5.py
mat = [
    [1, 0, 3],
    [0, 0, 4],
    [6, 0, 0]
]

m, n = len(mat), len(mat[0])
zero_count = sum(1 for i in range(m) for j in range(n) if mat[i][j] == 0)

if zero_count > (m * n) // 2:
    print("Yes")
else:
    print("No")
